designdocs/logical_batching.md

```diff
@@ -0,0 +1,67 @@
+# Summary
+
+In other documents, we explore ways to structure an explicit batch of messages.
+That is, in terms of concrete structure. This document has a different aim --
+deriving the idea of a batch logically, as a product of fields present on other
+entities in our system.
```

Comment:
So are we registering a Batch Message Schema using these fields?  Would that be the story to write from this document?

Reply:
I'm working on writing those stories. Here is the list I have so far to do:

- Story to add Format Type to the Schema (pointing to an enum with just Avro currently)
- Story to document Avro Format Type requirements
- Story to add Payload Location (on-chain only enum) to the Schema
  - Dependent-story to add the off-chain to the enum and support it in the message_pallet
- Story to add Parquet to the Format Type enum and document Parquet Format Type requirements

Reply:
https://github.com/LibertyDSNP/mrc/milestone/11

Reply:
This design doc needs to specify what fields are in a Batch message, including types.  If it's not settled then there's no Design Doc to write. If it's settled, it should be documented so that the stories can refer back to the Design Doc for implementation.

The Design Doc template is also part of the process, it's intended to help write the document.  Not all sections may need to be there (glossary or table of contents are two obvious examples)